### ISSUE TITLE
Improve quickupload

### ISSUE THREAD

**Issue by shylux** [4teamwork/opengever.core#422]
In addition to the styling in https://github.com/4teamwork/plonetheme.teamraum/pull/230 for `OGIP-7 OPT 7`.
- The quickuplod-box is now visible by default. It should be possible to close the box so it only appears if a file is dragged over.
  - Persistent with https://github.com/4teamwork/ftw.dictstorage
  - Add a close button. Current layout in `collective.quickupload`. https://github.com/collective/collective.quickupload/blob/master/collective/quickupload/browser/quick_upload.pt
    https://github.com/collective/collective.quickupload/blob/master/collective/quickupload/browser/static/fileuploader.js
    https://github.com/collective/collective.quickupload/blob/master/collective/quickupload/browser/quick_upload.py#L257
  - Functionality of close button with own `BrowserView`?
- Also make the font a little bit smaller
- Maybe rewording. Can be done by overwriting `collective.quickupload` translation.

**Comment by phgross** [comment]
Ist zurzeit kein Thema mehr. Ich schliesse diesen Issue.
